wavelength-plugins/factions.js

```diff
@@ -743,39 +743,34 @@ exports.commands = {
 			output += "</table></center>";
 			this.sendReplyBox(output);
 		},
-		'': 'help',
```

Comment:
you can keep these, just make them parse the help command now, its useful for finding commands on what to do.

Reply:
Wouldn't it be simplier just having it /factionhelp (works as /help faction)

Reply:
Done